pkg/flow/adapter/synchronizer/env.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package synchronizer
+
+import pkgadapter "knative.dev/eventing/pkg/adapter/v2"
+
+// EnvAccessorCtor for configuration parameters
+func EnvAccessorCtor() pkgadapter.EnvConfigAccessor {
+	return &envAccessor{}
+}
+
+type envAccessor struct {
+	pkgadapter.EnvConfig
+
+	CorrelationKey       string `envconfig:"CORRELATION_KEY"`
+	CorrelationKeyLength int    `envconfig:"CORRELATION_KEY_LENGTH"`
+	ResponseWaitTimeout  string `envconfig:"RESPONSE_WAIT_TIMEOUT"`
+
+	// BridgeIdentifier is the name of the bridge workflow this target is part of
+	BridgeIdentifier string `envconfig:"EVENTS_BRIDGE_IDENTIFIER"`
```

Comment:
the bridge identifier should be propagated as a CE attribute at outgoing events and responses.

Reply:
Setting BridgeID in outgoing events in 
https://github.com/triggermesh/triggermesh/pull/353/commits/c4b31e57f8f79488eb22f3b05b654f75d36e9bd8